ArcadiaTeamsBot/Startup.cs

```diff
@@ -16,7 +19,14 @@ public void ConfigureServices(IServiceCollection services)
             services.AddControllers();
 
             services.AddSingleton<IBotFrameworkHttpAdapter, BotAdapterWithErrorHandling>();
+

```

Comment:
The same here, it is allowed to not have such empty lines. Otherwise it looks unwieldy.

Reply:
Or you can group them logically. E.g.:
```cs
AddMediatr

AddHttpClient

AddScoped
AddScoped
AddSingleton
```

Reply:
Fixed